package.json

```diff
@@ -5,18 +5,21 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "build": "vuepress build docs",
+    "clean:dev": "vuepress dev docs --clean-cache",
     "dev": "vuepress dev docs",
     "format": "prettier ./**/*.md --write --ignore-path .gitignore",
     "lint": "remark . --ignore-path .gitignore",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "dep:update": "pnpm add @vuepress/client@next vuepress@next vuepress-theme-hope@next && pnpm i && pnpm up"
   },
   "keywords": [],
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "fontawesome-free": "^1.0.4",
```

Comment:
What is this package?
Any reason we aren't using the official package? [@fortawesome/fontawesome-free](https://www.npmjs.com/package/@fortawesome/fontawesome-free)

Reply:
Uh that's because I searched up font-awesome and it took me to that one. I can swap it if need be

Reply:
Yeah, we should use the proper one, no idea what that other one is.

Reply:
Should be fixed now